lib/components/molecules/StickyHeader/StickyHeader.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useRef, useEffect, Fragment } from 'react';
+import type, { Node } from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+import styles from './StickyHeader.style';
+import './StickyHeader.css';
+
+type Props = {
+  children: Node,
+};
+
+const StickyHeader = ({ children, className }: Props): Node => {
+  const [isSticky, setSticky] = useState(false);
+  const ref = useRef(null);
+  const handleScroll = () => {
```

Comment:
How do you handle the flicker or content that will slip below header since this is on z-index

Reply:
There could be external callback on handleScroll

Reply:
Flicker has been taken care using animation.